app/scripts/app.js

```diff
@@ -17,7 +23,8 @@ blocChat.config(['$stateProvider', '$locationProvider', function($stateProvider,
 
 blocChat.factory("chatRooms", [
     "$firebaseArray",
-    function($firebaseArray) {
+    "CurrentRoom",
+    function($firebaseArray, CurrentRoom) {
         var roomsUrl = 'https://stewart-bloc-chat.firebaseio.com/rooms';
         var roomsRef = new Firebase(roomsUrl);
         var roomsObject = $firebaseArray(roomsRef);
```

Comment:
citizens is hard coded because I want there to be a default room when the page loads. This obviously isn't extensible, but I'll figure out a different solution in a subsequent checkpoint.

good call on the root of the roomUrl. I'll clean DRY that up :)